package.json

```diff
@@ -2,7 +2,7 @@
   "name": "snoozetabs",
   "description": "An add-on to let you snooze your tabs for a while.",
   "id": "snoozetabs@mozilla",
-  "version": "1.0.0",
+  "version": "1.0.9",
```

Comment:
Pending a fix for #110, we need to update this version in both `package.json` and `manifest.json` to ensure the deploy process works. AMO will throw an error on signing if we attempt a push to `production` branch with a previously-signed version number - which is why we had to increment through v1.0.0 to v1.0.9 while troubleshooting 😞